tornettools/generate_tgen.py

```diff
@@ -388,12 +447,17 @@ def __get_tgen_clients(args, n_users, n_circuits, measurement2):
         exponential_rate = 1.0 / usec_per_circ
 
         client = {
-            'name': 'markovclient{}'.format(i+1),
             'circuit_rate_exp': exponential_rate,
             'country_code': chosen_country_code,
             'log_level': 'info' if i == 0 else 'message',
+            'is_hs_client': is_hs_client,
         }
 
+        if is_hs_client:
+            client['name'] = 'markovclient{}onionservice'.format(i+1)
+        else:
+            client['name'] = 'markovclient{}'.format(i+1)
```

Comment:
`markovclient{}public`?

Reply:
Renamed the `markovclient{}` hosts to `markovclient{}public`.